### PR TITLE
Vanderburg animation fixes1

### PR DESCRIPTION
**Pull request by brianvanderburg2**
This provides two fixes to the animation code:
* Normally, the first frame is not entered. Upate_Animation only calls Enter after it leaves one frame and changes to the next.  This means with an array of ground sprites using image sets with random times, the first frame will display the minimum time instead of a random time.  The fix I used is to call Enter for the first frame inside of Set_Image_Set
* The next change makes it where an image set can reference files anywhere along the package search path instead of relative to the location of the image set file.  I could create a package, then add an image set under my pixmaps path somewhere, but reference images directly in the base data without having to copy the files into my package.  The commit comment explains it best.

Basically, if my image set file is /package1/pixmaps/something/my.imgset, then it is treated relatively as something/my.imgset.  Then if an image reference is used such as ../blocks/ice/1.png, it gets combined as something/../blocks/ice/1.png, then normalized as blocks/ice/1.png, so the package manager can use any path to find blocks/ice/1.png

### CODE REVIEW

**Comment by Quintus** [comment]
You don’t have to file PRs if you only improve existing stuff unless you want the team to review your changes for some specific reason. Reserve PR’s for the big, important things (and those that need discussion or licensing confirmation). Just merge your branch into `devel` without asking in such cases.

Apart from that, looks fine to me.

Vale,
Quintus
